### PR TITLE
Follow REUSE specification

### PR DESCRIPTION
**Pull request by felixfontein**
https://reuse.software/spec/

### CODE REVIEW

**Comment by felixfontein** [comment]
CC @gotmax23

File: src/antsibull_core/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# coding: utf-8
```
Comment (gotmax23) [inline src/antsibull_core/__init__.py:1]: [The default encoding for Python 3 is already `utf-8`](https://peps.python.org/pep-3120/). I don't think this is necessary.

Comment (gotmax23) [reply src/antsibull_core/__init__.py:1]: /me pushed the wrong button again...

File: LICENSES/PSF-2.0.txt [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+
+Starting with Python 3.8.6, examples, recipes, and other code in
+the documentation are dual licensed under the PSF License Version 2
+and the Zero-Clause BSD license.
+
+Some software incorporated into Python is under different licenses.
+The licenses are listed with code falling under that license.
+
+
+PYTHON SOFTWARE FOUNDATION LICENSE VERSION 2
```
Comment (gotmax23) [inline LICENSES/PSF-2.0.txt:73]: Only this part is the `PSF-2.0` license, which is what applies to the argparse code.

File: src/antsibull_core/__init__.py (hunk 1 above)
Comment (felixfontein) [reply src/antsibull_core/__init__.py:1]: That would have applied to antsibull as well :-)

File: src/antsibull_core/vendored/json_utils.py [hunk 3]
```diff
@@ -1,28 +1,7 @@
...
-# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
-# IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
-# INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
-# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
-# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
-# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
-# USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
```
Comment (gotmax23) [inline src/antsibull_core/vendored/json_utils.py:2]: I believe this is supposed to be the BSD version?

**Review by gotmax23 — COMMENTED** [review]
These are my initial comments. I may find something else once I take a closer look ;).

File: src/antsibull_core/__init__.py (hunk 1 above)
Comment (felixfontein) [reply src/antsibull_core/__init__.py]: Implemented in b937bee5facc0faf680bba83792e536b26fa9b40.

File: LICENSES/PSF-2.0.txt (hunk 2 above)
Comment (felixfontein) [reply LICENSES/PSF-2.0.txt:1]: Fixed in dbe629d2717904e086506fade3da2a9dc0765bb5.

File: src/antsibull_core/vendored/json_utils.py (hunk 3 above)
Comment (felixfontein) [reply src/antsibull_core/vendored/json_utils.py]: Fixed in dbe629d2717904e086506fade3da2a9dc0765bb5.

**Review by gotmax23 — APPROVED** [review]

File: LICENSES/BSD-2-Clause.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+
+2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
```
Comment (gotmax23) [inline LICENSES/BSD-2-Clause.txt:8]: Trailing newline.

File: src/antsibull_core/vendored/_argparse_booleanoptionalaction.py [hunk 5]
```diff
@@ -3,6 +3,9 @@
 # It is licensed under the Python Software Foundation License:
 # https://github.com/python/cpython/blob/main/LICENSE
 #
+# SPDX-FileCopyrightText: 2001-2020 Python Software Foundation
```
Comment (gotmax23) [inline src/antsibull_core/vendored/_argparse_booleanoptionalaction.py:6]: Where does this copyright date come from? It's different than the one in the PSF license file.

**Comment by gotmax23** [comment]
> @felixfontein
[Strip unnecessary content from the PSL-2.0.](https://github.com/ansible-community/antsibull-core/pull/5/commits/dc38752d5329c37f2ded709c20ef8b5255468712)

is misspelled, but the commit will be squashed in the end, so meh. It shouldn't be to hard to `git rebase -i HEAD^^` and reword the commit message, though.

File: LICENSES/BSD-2-Clause.txt (hunk 4 above)
Comment (felixfontein) [reply LICENSES/BSD-2-Clause.txt]: Thanks, fixed in 26ac52d32b206251be0118442b61e01981d312cd.

File: src/antsibull_core/vendored/_argparse_booleanoptionalaction.py (hunk 5 above)
Comment (felixfontein) [reply src/antsibull_core/vendored/_argparse_booleanoptionalaction.py:6]: The code was last modified in 2020 (https://github.com/python/cpython/commit/41b223d29cdfeb1f222c12c3abaccc3bc128f5e7).

Comment (gotmax23) [reply src/antsibull_core/vendored/_argparse_booleanoptionalaction.py:6]: :+1:

**Review by gotmax23 — APPROVED** [review]

**Comment by felixfontein** [comment]
@gotmax23 thanks for reviewing this!

**Pull request merged**